common/components/org.wso2.carbon.data.agents.log4j/src/main/java/org/wso2/carbon/data/agents/log4j/appender/LogEventAppender.java

```diff
@@ -0,0 +1,411 @@
+/*
```

Comment:
This class seems highly inefficient and not so stable in connecting DAS instance. Need to put a JIRA and fix it immediately.